### ISSUE TITLE
Error when running composer install --no-dev

### ISSUE THREAD

**Issue by AidanLaycock** [aarondfrancis/solo#28]
When running `composer install --no-dev` you receive the following error message:

```
  Class "AaronFrancis\Solo\Providers\SoloApplicationServiceProvider" not found
```

Which prevents the application from building. I'm not sure if a Deferred Provider (https://laravel.com/docs/11.x/providers#deferred-providers) would be a solution for this? Or if I'm missing a more obvious solution. If you know generally how to fix this, then I'd be happy to get a pointer or two and then to contribute to the package as it's already been massively useful in local development! 🙌

**Comment by aarondfrancis** [comment]
I think that's fixed in 0.2.1! Seems like the same thing as #19

**Comment by AidanLaycock** [comment]
Amazing, yes that works now for me! I did need to re-install solo but I think it's a niche issue there 😄 Thanks for the quick response.
